src/store/lookup.ts

```diff
@@ -35,7 +35,8 @@ async function lookup(browser: Browser, store: Store) {
 			continue;
 		}
 
-		const page = await browser.newPage();
+		const context = await browser.createIncognitoBrowserContext();
+		const page = (config.browser.isIncognito ? await context.newPage() : await browser.newPage());
```

Comment:
I'd wrap this context within the if statement as well, so we don't create the incognito browser context without needing to.